features/freaks/src/main/kotlin/com/agilefreaks.freaks_catalog.features.freaks/FreaksFragment.kt

```diff
@@ -83,4 +98,43 @@ class FreaksFragment : Fragment() {
             repeat(FREAKS_COUNT) { this.add(freak) }
         }
     }
+
+    private fun loadFilters(activeFilter: String) =
+        if (activeFilter == "Skills") {
```

Comment:
What happens if by mistake instead of "Skills" I pus "Skill"? Because I am just a simple person who can always make keyboard mistakes

Reply:
No one is actually typing what kind of filter they want to see. We pass a string with the filter name so we can also set the Filter Title when the bottom sheet dialog shows up. I will think of better ways to do this anyway.